pyanaconda/display.py

```diff
@@ -406,7 +406,9 @@ def on_mutter_ready(observer):
                     "an RDP session to connect to this computer from another computer and "
                     "perform a graphical installation or continue with a text mode "
                     "installation?")
-        rdp_creds = ask_rd_question(anaconda, message)
+        # we aren't really interested in the use_rd flag so at least mark it like this
+        # to avoid linters being grumpy
+        _use_rd, rdp_creds = ask_rd_question(anaconda, message)
```

Comment:
Honestly to me this doesn't seem correct.

I don't think we should set the `flag.use_rd`, it looks to me like a side effect. Another option is to not return the value but that would be less preferable to me as the side effect is still there.

In other words, from looking on this change set it doesn't sense why we ignore this value when we ask for it on line below, however, it's not visible that the `use_rd` flag is set inside the `ask_rd_question`.

Reply:
I would rather rewrite the calls in this method to something like:
```python
def set_gui_mode_on_rdp(anaconda, use_rd):
        if not anaconda.gui_mode:
            log.info("RDP requested via RDP question, switching Anaconda to GUI mode.")
        anaconda.display_mode = constants.DisplayModes.GUI
        flags.use_rd = True

use_rd, rdp_creds = ask_rd_question(message)
set_gui_mode_on_rdp(anaconda, use_rd)
```

Seems cleaner to me.

Reply:
Sounds good - yeah, it might actually make sense to set the flag there as well - its should still be marked as using RD even after Wayland crashed. Must have somehow mixed it all up with GUI/non-GUI mode. :)

Feel free to open a new PR & we can then close this one as obsolete. :)